app/cypress/e2e/SecvisogramPage/SecureHref.cy.js

```diff
@@ -0,0 +1,15 @@
+import { expect } from 'chai'
+import HTMLTemplate from '../../../lib/app/SecvisogramPage/View/shared/HTMLTemplate.js'
+import DocumentEntity from '../../../lib/app/shared/Core/entities/DocumentEntity.js'
+import secureHrefTests from '../../fixtures/secureHrefTests.js'
+
+describe('secureHrefTests', () => {
+  secureHrefTests.forEach((test, i) => {
+    it(test.title ?? `Test #${i + 1}`, () => {
+      const preview = new DocumentEntity().preview({ document: test.content })
+      const html = HTMLTemplate(preview)
+      const hasHref = html.match(/<a href=".*test\.org.*">.*<\/a>/)
```

Comment:
Please use example.com instead - test.org is IMHO not reserved for test purposes...
```suggestion
      const hasHref = html.match(/<a href=".*example\.com.*">.*<\/a>/)